extensions/extensions-helm/aissemble-spark-history-chart/examples/values-s3.yaml

```diff
@@ -0,0 +1,19 @@
+eventVolume:
+  enabled: false
+
+dependencies:
+  packages:
+    - org.apache.hadoop:hadoop-aws:3.3.4
+
+deployment:
```

Comment:
Would recommend mirroring the env methods supported on operator: https://github.com/kubeflow/spark-operator/blob/master/docs/user-guide.md#specifying-environment-variables

Reply:
Yea that makes sense. What about including these options though:
```
    envFrom:
      - configMapRef:
          name: env-config-map
      - secretRef:
          name: env-secret
```

Reply:
Where you can reference a secret/config map as a whole

Reply:
I think that's a reasonable ask.  This PR is already going to be massive, though-- I'd rather tackle that as separate follow-ons.